runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/client/grpc/GrpcWindmillStreams.java

```diff
@@ -62,7 +62,7 @@
  */
 @ThreadSafe
 @Internal
-public class GrpcWindmillStreamFactory implements StatusDataProvider {
+public class GrpcWindmillStreams implements StatusDataProvider {
```

Comment:
Why rename? It is like a factory that creates streams where as GrpcWindmillStreams doesn't convey what it does.  The variable and parameter names throughout still refer to it as a factory and changing those to remove factory would make it less clear what it is.

Reply:
I think GrpcWindmillStreams is better than GrpcWindmillStreamFactory, since this class is not merely a factory creating instances, but it also keeps track of the created instances and  does health checks.

Reply:
reverted